gradle/README.md

```diff
@@ -0,0 +1,6 @@
+# Gradle
+
+To regenerate the [project-schema.json](project-schema.json) file run this in the project's root directory:
+```bash
+./gradlew kotlinDslAccessorsSnapshot
```

Comment:
This doesn't explain when or why you would do this though, or what the project-schema document is....